### PR TITLE
Added input fields for collecting emails and performed email validation over user input

### PR DESCRIPTION
**Pull request by kaustubhpDev**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -75,7 +75,7 @@ footer {
   }
 }
 
-.cta-button:hover {
+.cta-button,.block:hover {
```
Comment (akankshadharkar) [inline css/main.css:78]: I can't really see the block selector used anywhere and have you checked if removing the :hover from the cta-button causes any unexpected change in the UI?

NIT: Can you please add a space between the two selectors' separator (.cta-button, .block:hover)

File: index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
   <title>Real Dev Squad</title>
   <link rel="stylesheet" href="./css/main.css">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```
Comment (akankshadharkar) [inline index.html:11]: @ankushdharkar are we sure about starting the use of bootstrap?

File: index.html [hunk 3]
```diff
@@ -17,11 +18,17 @@ <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
     <h2 class="info-note highlighted-text">You can check out all our current events happening <a href="./events.html">here</a></h2>
     <img class="logo-img" src="img/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
     <h3 class="info-note"> To know how this page is getting developed, join our Discord Channel. Contact one of the existing members </h3>
+    <div class="cta-button__container " id = "buttonDiv">
+      <h4 align="center">Please subscribe for further event updates</h4>
```
Comment (akankshadharkar) [inline index.html:22]: AFAIK: 
HTML align attribute supports col, colgroup, tbody, td, tfoot, th, thead, tr elements.
Usage of align attribute for any other HTML elements is deprecated; You must use CSS for those

Please confirm

File: index.html [hunk 4]
```diff
@@ -17,11 +18,17 @@ <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
...
+    </div>
   </main>
 
   <footer>
     <p class="info-repo">The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website" target="_blank" rel="noopener noreferrer">open sourced repo</a></p>
   </footer>
-
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
```
Comment (akankshadharkar) [inline index.html:31]: @ankushdharkar just checking if we're using jquery and I'm not aware about that discussion

File: subscribeButton.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+// Creating a button Element
+const div = document.createElement('div');
```
Comment (akankshadharkar) [inline subscribeButton.js:2]: If the subscribe button will always be there (the div), why are we not creating the div in html itself?

File: subscribeButton.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+<form>
+<input type="text" class="form-control " id="email" placeholder="Please Enter Email ">
+              <div class="invalid-feedback">
+                Enter a valid email
+              </div>        
+              <br>
+ <input type="submit" value="Subscribe" class="btn btn-primary "> 
+           
+</form>
+</div>
```
Comment (akankshadharkar) [inline subscribeButton.js:15]: Same question

File: subscribeButton.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+ `;
+const buttonDiv = document.getElementById('buttonDiv');
+buttonDiv.style.padding="100px 100px";
```
Comment (akankshadharkar) [inline subscribeButton.js:18]: CSS class selectors on the html tag would be better

**Review by swarajpure — COMMENTED** [review]
@kaustubhpDev What's up with this PR?

**Pull request closed without merging**